components/eventing-controller/controllers/subscriptionv2/reconcilertesting/test_utils.go

```diff
@@ -80,11 +82,17 @@ func EventuallyUpdateSubscriptionOnK8s(ctx context.Context, ens *TestEnsemble,
 	}, SmallTimeout, SmallPollingInterval).ShouldNot(gomega.HaveOccurred(), "error while updating the subscription")
 }
 
-func CreateSubscription(ens *TestEnsemble, subscriptionOpts ...v2.SubscriptionOpt) *eventingv1alpha2.Subscription {
+func GetSubscription(ens *TestEnsemble, subscriptionOpts ...v2.SubscriptionOpt) *eventingv1alpha2.Subscription {
```

Comment:
(nit) By `GetSubscription` I thought it will fetch the subscription from k8s. Can we have some clear name? like `NewSubscription` etc.

Reply:
yes will do

Reply:
done